src/main/resources/reference.conf

```diff
@@ -1174,6 +1174,35 @@ resourceTypes = {
     }
     reuseIds = true
   }
+  datarepo-google-project = {
+    actionPatterns = {
+      read_policies = {
+        description = "list all policies and policy details for this resource"
+      }
+      delete = {
+        description = "delete this resource"
+      }
+      get_parent = {
+        description = "get the parent resource of this resource"
+      }
+      set_parent = {
+        description = "set the parent resource of this resource"
+      }
+      link = {
+        description = "link this resource to a dataset or datasnapshot"
+      }
```

Comment:
In #1652, we added a `link` action to `google-project` to control the ability to set/remove billing from the underlying Google project. Is this `link` action suitable for that as well? If not, we'll need a new action for billing.